### ISSUE TITLE
Suggesting some features for this add on

### ISSUE THREAD

**Issue by TruongQToan** [glutanimate/review-heatmap#26]
I'd like to thank you about this add-on, after I know this I have more motivation to learn.

After using for awhile, I have some new ideas about this addon.
- Add how much time I spend per day for Anki (hover a day in heatmap to reveal)
- Longest streak for days of learning at least ??? minus (??? user define this - default is 1 minus

**Comment by glutanimate** [comment]
Thanks, Truong! I appreciate the suggestions. The first one sounds good, especially.

I'm not 100% sure what you mean by the second one, however. Would you just like to subtract a configurable number of days from the longest streak? If so, could you describe your use case for that?

Thanks again :)

**Comment by picsi** [comment]
I was going to suggest number one too.

It would be useful to have the hovertext say: "100 cards in 20 minutes on Friday, December 11, 2012"

**Comment by issac-temp** [comment]
I just wanted to input what I am sure the OP meant with the second idea:
Have an option to not count a day of learning, unless you study for more than X amount of minutes (I think his "minus" was meant as "minutes").

And I have to agree with that suggestion, it would be really neat to have an "anti-cheat" option so that you can't just study one card to keep the streak going.

**Comment by glutanimate** [comment]
Thanks for chiming in, @issac-temp. That definitely makes sense. Though I'm not sure whether the approach in #7 wouldn't be a better solution in terms of an "anti-cheat" option.

**Comment by issac-temp** [comment]
Yes, I just checked #7 out and thats definitely a better solution for that "anti-cheat" option. Thanks!
